history/helpers.js

```diff
@@ -0,0 +1,150 @@
+let { splitTags } = require('../config');
+
+const propMap = {
```

Comment:
Isn't it somehow possible to reuse [this](https://github.com/softwaregroup-bg/ut-rule/blob/a64ed0a7dd784217e5c2d1cae8e8e1fee7c78ac9/ui/react/pages/RuleProfile/helpers.js#L219) code? Duplicating 150 lines of code with just slight differences doesn't look like a good idea.

Reply:
Yes similar type of code exists in front end code with some differences, this above one is used at back end.  I will make it an a common place and refer  it at required places

Reply:
@stambata  Can you have look at now ?